### PR TITLE
add bar height parameter

### PR DESCRIPTION
**Pull request by entonbiba**
add barHeight paramter, I'll update the docs once this gets merged.
#915 #944

example: http://codepen.io/entonbiba/pen/VpedWZ

### CODE REVIEW

**Comment by mspae** [comment]
Awesome!

Currently a higher value produces a larger waveform and a lower value a smaller one. Maybe we could change the name of the parameter, otherwise it will be confusing. (Something like `shrinkage` :grinning:)

**Comment by entonbiba** [comment]
ok I made it accept high numbers :P for example `100` makes it huge, while 0.5 smaller.

File: src/drawer.multicanvas.js [hunk 1]
```diff
@@ -220,7 +220,7 @@ WaveSurfer.util.extend(WaveSurfer.Drawer.MultiCanvas, {
         var offsetY = height * channelIndex || 0;
         var halfH = height / 2;
 
-        var absmax = 1;
+        var absmax = 1/this.params.barHeight;
```
Comment (thijstriemstra) [inline src/drawer.multicanvas.js]: spaces around operator

Comment (entonbiba) [inline src/drawer.multicanvas.js]: @thijstriemstra updated

**Pull request merged**